browser-interface/packages/shared/realm/resolver.ts

```diff
@@ -71,6 +71,10 @@ export function prettyRealmName(realm: string, candidates: Candidate[]) {
   return realm
 }
 
+export function isEns(str: string | undefined): str is `${string}.eth` {
+  return !!str?.match(/^[a-zA-Z0-9]+\.eth$/)?.length
```

Comment:
This will not support ens subdomains like `nando.company.eth`

Reply:
Doing that will probably make it unnecessary to use isDclEns on other places as it will pick .dcl.eth names as well

Reply:
Thanks @fzavalia. Much nicer and simpler.